inject/README.md

```diff
@@ -1,6 +1,20 @@
-# DInject
+# Avaje Inject
 APT based dependency injection for server side developers - https://dinject.io
 
+### Example module use
+
+```java
+module org.example {
+
+  requires javax.inject;
+  requires org.slf4j;
+  requires transitive io.avaje.inject;
```

Comment:
Just curious as to why it has to be transitive?

Reply:
Nice catch, it doesn't have to be.  Fixing and updating this.